src/main/flight/mixer.c

```diff
@@ -473,6 +482,20 @@ static void applyMixToMotors(const float motorMix[MAX_SUPPORTED_MOTORS], motorMi
             motor[i] = motor_disarmed[i];
         }
     }
+
+#ifdef USE_WING
+    float motorSumSquares = 0.0f;
+    if (motorIsEnabled()) {
+        for (int i = 0; i < mixerRuntime.motorCount; i++) {
+            if (motorIsMotorEnabled(i)) {
+                const float motorOutput = scaleRangef(motorConvertToExternal(motor[i]), PWM_RANGE_MIN, PWM_RANGE_MAX, 0.0f, 1.0f);
+                motorSumSquares += motorOutput * motorOutput;
+            }
+        }
+    }
+    rootMeanSquareMotorOutput = sqrt(motorSumSquares / mixerRuntime.motorCount);
```

Comment:
```suggestion
    rootMeanSquareMotorOutput = sqrtf(motorSumSquares / mixerRuntime.motorCount);
```

Reply:
Isn't there gcc warning for double math? Or it is only for double constants?

Reply:
yeah im not getting any warnings if i use sqrt instead of sqrtf

Reply:
Sorry to miss this one. @limonspb maybe check this in your announced follow up PR's

Reply:
Also see: https://stackoverflow.com/questions/7431040/difference-between-sqrt-and-sqrtf

Reply:
TL:DR using sqrt works with floats as sqrtf works also using double, long double and float

Reply:
Anyway could update in follow PR

Reply:
> Anyway could update in announced followup PR's if needed.

yeah i changed to `sqrtF` in this PR :)

Reply:
Most of SO answer is C++ only. In C, argument must be converted to `double`, then `sqrt` is calculated, using software implementation of `double`. This will be considerably more expensive than 14cycle VSQRT inctruction.